diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter6/CustomGetterSetterRule.kt

```diff
@@ -33,6 +35,21 @@ class CustomGetterSetterRule(configRules: List<RulesConfig>) : DiktatRule(
         val isPrivateSetter = node.getFirstChildWithType(MODIFIER_LIST)?.hasAnyChildOfTypes(PRIVATE_KEYWORD) ?: false
         val isOverrideGetter = node.treeParent.getFirstChildWithType(MODIFIER_LIST)?.hasAnyChildOfTypes(OVERRIDE_KEYWORD) ?: false
 
+        // find matching node
+        val propertyList = node.treeParent?.treeParent?.getAllChildrenWithType(PROPERTY)
+        val nodeName = node.treeParent?.getFirstChildWithType(IDENTIFIER)
+        nodeName?.let {
```

Comment:
you have a lambda over a `nodeName`, but it is not used as `it` directly (uni are using it at line 46 directly)

Reply:
Changed